### PR TITLE
currentColor: add definition and links

### PR DESCRIPTION
**Pull request by estelle**
currentColor is used, but the definition is nearly impossible to find, and many people don't know what it is. So, surfacing it. Also, it has special meaning in `color`, so defining it there.

Also, note I used `currentColor` and not `currentcolor` intentionally. Setting camel case enables screen readers to read it in a more helpful way. The spec has it lowercase, as do the rest of the docs, so am asking screen reader users what they prefer.

Is this similar to the rule to always remember to #camelCaseHashTagsWhereNormallyThereAreWordBreaks? Not sure.

DRAFT mode while waiting for feedback from the a11y folks I asked. But if there was already a discussion about this, please let me know.

### CODE REVIEW

File: files/en-us/web/css/color/index.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ Note that the value must be a uniform {{cssxref("color")}}. It can't be a {{cssx
 
 - {{cssxref("&lt;color&gt;")}}
   - : Sets the color of the textual and decorative parts of the element.
+- [`currentColor`](/en-US/docs/Web/CSS/color_value#currentcolor_keyword)
+  - : Normally sets the color to the element's `color` property value, but when set as the value of `color` it is treated as `inherit`.
```
Comment (dipikabh) [inline files/en-us/web/css/color/index.md:88]:
```suggestion
  - : Sets the color of the element's `color` property value. However, if set as the value of `color`, `currentColor` is treated as `inherit`.
```

should 'to' be 'of'?

**Review by dipikabh — CHANGES_REQUESTED** [review]
I took a look even though you have it in DRAFT mode. The two mentions of 'currentcolor' (all lowercase) in the first para (line 24) would also need to be updated?

File: files/en-us/web/css/color/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/color/index.md:88]: should be "to".

File: files/en-us/web/css/color/index.md [hunk 2]
```diff
@@ -84,6 +84,8 @@ Note that the value must be a uniform {{cssxref("color")}}. It can't be a {{cssx
 
 - {{cssxref("&lt;color&gt;")}}
   - : Sets the color of the textual and decorative parts of the element.
+- [`currentcolor`](/en-US/docs/Web/CSS/color_value#currentcolor_keyword)
+  - : Sets the color to the element's `color` property value. However, if set as the value of `color`, `currentColor` is treated as `inherit`.
```
Comment (estelle) [inline files/en-us/web/css/color/index.md:88]:
```suggestion
  - : Sets the color to the element's `color` property value. However, if set as the value of `color`, `currentcolor` is treated as `inherit`.
```

**Comment by estelle** [comment]
I got two bits of feedback:

> `PascalCase` or `camelCase` always help me when strings are smooooshed. i prefer `camelCase`.

and

> Interesting question.  I turned on NVDA to read your post, and it announced them exactly the same (no difference to my ear).

so i think we need to further discuss, and not make the change starting with this PR ;)

**Review by dipikabh — APPROVED** [review]

**Pull request merged**